app/views/rules_of_origin/_tab.html.erb

```diff
@@ -5,7 +5,7 @@
       <%= country_flag_tag country_code, alt: "Flag for #{country_name}" %>
     </h2>
 
-    <% if rules_of_origin_schemes.present? %>
+    <% if rules_of_origin_schemes.any? %>
```

Comment:
Yeh - in this instance its operating on an array _but_ `#present?` checks for `#nil?` first and tries to be clever - `#any?` will only work if data is an enumerable - this always should be one, if its not then it'll raise which is good

Reply:
Re test coverage - good shout but just been looking over the spec and it does have coverage for single, multiple and none scheme scenarios so I _think_ we're good

I have found there's a couple of 'old' specs for the old legacy tab still in there - they are passing because they check for absence rather then presence so I've adding a commit dropping those